src/core_plugins/tile_map/public/editors/tile_map.html

```diff
@@ -13,69 +13,6 @@
 <div ng-if="vis.params.mapType === 'Heatmap'" class="form-group">
   <div>
     <label>
-      Radius
-      &nbsp;<kbn-info placement="right" info="Size of heatmap dots. Default: 25"></kbn-info>
-    </label>
-    <div class="vis-editor-agg-form-row">
-      <input
-        name="heatRadius"
-        ng-model="vis.params.heatRadius"
-        required
-        class="form-control"
-        type="range"
-        min="5"
-        max="50"
-        step="1"
-        >
-      <div class="form-group vis-editor-agg-form-value">
-        {{vis.params.heatRadius}}
-      </div>
-    </div>
-  </div>
-  <div>
-    <label>
-      Blur
-      &nbsp;<kbn-info placement="right" info="Amount of blur applied to dots. Default: 15"></kbn-info>
-    </label>
-    <div class="vis-editor-agg-form-row">
-      <input
-        name="heatBlur"
-        ng-model="vis.params.heatBlur"
-        required
-        class="form-control"
-        type="range"
-        min="1"
-        max="25"
-        step="1"
-        >
-      <div class="form-group vis-editor-agg-form-value">
-        {{vis.params.heatBlur}}
-      </div>
-    </div>
-  </div>
-  <div>
-    <label>
-      Maximum zoom
-      &nbsp;<kbn-info placement="right" info="Map zoom at which all dots are displayed at full intensity. Default: 0"></kbn-info>
-    </label>
-    <div class="vis-editor-agg-form-row">
-      <input
-        name="heatMaxZoom"
-        ng-model="vis.params.heatMaxZoom"
-        required
-        class="form-control"
-        type="range"
-        min="1"
-        max="18"
-        step="1"
-        >
-      <div class="vis-editor-agg-form-value">
-        {{vis.params.heatMaxZoom}}
-      </div>
-    </div>
-  </div>
-  <div>
-    <label>
       Minimum opacity
```

Comment:
I'd label this as something more wishy-washy: 'Granularity', 'Intensity'. The reason for that is that it ties us less to the L.heatlayer implementation and would make it easier to backfill this generic property with another heat-implementation.  

Range form 0-1 remains good though.